### PR TITLE
Fix Lim Launched RC medium half loops from clipping in to eachother 

### PR DESCRIPTION
**Pull request by mixiate**
This fixes the lim launched rollercoaster medium half loops from clipping in to eachother. In order for the LIM train to go through these loops correctly I have changed the draw order 9 bounding boxes for pitch up 105 and 120. This brings them more in line with the bounding boxes of other trains/vehicles.

https://github.com/user-attachments/assets/dd2f6bbd-5344-4400-92ea-d7e1f57fda27

https://github.com/user-attachments/assets/ff0b37a8-f469-429b-9950-98999fd37d61

### CODE REVIEW

**Comment by Gymnasiast** [comment]
@SpartanFrederic104 this one too

**Comment by SpartanFrederic104** [comment]
Finally got around to testing. There is still some minor Z-fighting like with the other PR I tested but again that's more an issue with the trains themselves (yes, all three LIM coaster train types). In fact I can even see the LIM coaster trains Z-fighting on straight vertical track. I am using the software drawing engine for this, so there might be some different spots for Z-fighting on OpenGL.
Fred's Workbench 2024-10-29 16-12-06
Fred's Workbench 2024-10-29 16-13-40
Other than the minor Z-fighting issues from the trains themselves, the LIM medium half loops no longer clip into each other when built back-to-back. Looks great.

File: distribution/changelog.txt [hunk 1]
```diff
@@ -16,6 +16,7 @@
 - Fix: [#23044] "remove_unused_objects" command causes blank peep names.
 - Fix: [#23048] Map generator allows map sizes out of range through text input.
 - Fix: [#23058] [Plugin] Changing window colours doesn’t trigger the window to be fully redrawn.
+- Fix: [#23085] LIM Launched Roller Coaster medium half loops clip in to eachother.
```
Comment (SpartanFrederic104) [inline distribution/changelog.txt:19]: Should be:
``- Fix: [#23085] LIM Launched Roller Coaster medium half loops clip into each other when built back-to-back.``

Comment (mixiate) [reply distribution/changelog.txt]: Thanks, i've changed it to this.

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**